### PR TITLE
Fix #1368: Modify datatype of pa_activation.extras

### LINKED ISSUE #1368: Modify datatype of pa_activation.extras
pa_activation.extras is varchar(255) prior to 1.7.0. we need to enlarge it for additional FIDO2 information to 4000 chars.

Fix current Liquibase increment `20240115-add-columns-fido2.xml` where the column is modified to TEXT as this change is impossible by simple alter on Oracle DB.

User varchar(4000) instead of TEXT.
Modify retrospectively `20240115-add-columns-fido2.xml` as it is currently producing a invalid increment on Oracle DB

### PR DESCRIPTION
**Pull request by jandusil**

### CODE REVIEW

File: docs/db/changelog/changesets/powerauth-java-server/1.7.x/20240115-add-columns-fido2.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
     </changeSet>
 
     <changeSet id="3" logicalFilePath="powerauth-java-server/1.7.x/20240115-add-columns-fido2" author="Roman Strobl">
-        <modifyDataType tableName="pa_activation" columnName="extras" newDataType="TEXT"/>
+        <modifyDataType tableName="pa_activation" columnName="extras" newDataType="varchar(4000)"/>
```
Comment (banterCZ) [inline docs/db/changelog/changesets/powerauth-java-server/1.7.x/20240115-add-columns-fido2.xml:49]: cc @korbelm checksum has changed

**Review by banterCZ — COMMENTED** [review]
Missing edit in `PowerAuth-Server-1.7.0.md`.

**Review by banterCZ — APPROVED** [review]
Looks good now.

File: docs/PowerAuth-Server-1.7.0.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Following columns have been added to table `pa_activation` for FIDO2 support:
 - `external_id` - external identifier of the activation
 - `protocol` - protocol enumeration: `powerauth` or `fido2`
 
-The data type for column `extras` in table `pa_activation` was changed to `TEXT` / `CLOB` to support larger data.
+The data type for column `extras` in table `pa_activation` was changed to `VARCHAR` / `VARCHAR2` to support larger data.
```
Comment (zcgandcomp) [inline docs/PowerAuth-Server-1.7.0.md:21]:
```suggestion
The size of column `extras` in table `pa_activation` was increased to `4000` from `255` to support larger data.
```

**Review by zcgandcomp — CHANGES_REQUESTED** [review]

**Review by zcgandcomp — COMMENTED** [review]
Looks OK.

Checksums - no need to add them. And I belive @korbelm was notified.

**Review by zcgandcomp — APPROVED** [review]

**Pull request merged**